src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx

```diff
@@ -278,13 +317,22 @@ class Vivify extends Analyzer {
         }
       });
     }
-    const rems = vivifyHits.length - 1;
+    invigoratingMistHits.forEach((event) => {
+      const effective = event.amount + (event.absorbed || 0);
+      if (effective === 0) {
+        fullOverhealHits += 1;
+      }
+    });
+    const overheal = overhealPerCast / (healingPerCast + overhealPerCast);
```

Comment:
```suggestion
    const percentOverheal = overhealPerCast / (healingPerCast + overhealPerCast);
```
lets use a clearer variable name